src/bp/ui-admin/src/Pages/Workspace/Bots/index.tsx

```diff
@@ -292,7 +317,7 @@ class Bots extends Component<Props> {
   }
 
   get isPipelineView() {
-    return this.props.workspace.pipeline.length > 1
+    return this.props.workspace && this.props.workspace.pipeline && this.props.workspace.pipeline.length > 1
```

Comment:
We really need optional chaining on ui-admin, I think it's the only place that doesn't support it right now